appshell/cefclient_mac.mm

```diff
@@ -334,8 +334,15 @@ - (void)windowWillEnterFullScreen:(NSNotification *)notification {
     }
     if (customTitlebar) {
         [customTitlebar setHidden:YES];
+        NSWindow *window = [notification object];
+
+        // Since we have nuked the title, we will have
+        // to set the string back as we are hiding the
+        // custom title bar.
+        [window setTitle:[customTitlebar titleString]];
```

Comment:
About the deprecation and 10.6 target, I think that is something we can work on post 1.7. Agree with you updating the target as well as the removing the npm deprecation warnings.